keps/sig-api-machinery/1904-efficient-watch-resumption/kep.yaml

```diff
@@ -13,6 +13,7 @@ approvers:
   - "@deads2k"
   - "@lavalamp"
 prr-approvers:
+  - "@deads2k"
```

Comment:
@deads2k - please review in the already committed file:
https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/1904-efficient-watch-resumption#production-readiness-review-questionnaire

[I will update if needed]

Reply:
> @deads2k - please review in the already committed file:
> https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/1904-efficient-watch-resumption#production-readiness-review-questionnaire
> 
> [I will update if needed]

you owe a manual upgrade/downgrade/upgrade test this release.

Reply:
Yes - I will do that (though may not get to it before feature freeze before when this KEP needs to be merged).
Will send a PR with KEP update when this is done.